diagrams/onprem/ceph.py

```diff
@@ -0,0 +1,15 @@
+# This module is automatically generated by autogen.sh. DO NOT EDIT.
```

Comment:
This file seems unused file.

Reply:
Wait a minute.

Reply:
Any progress :)?

Reply:
I'm not finding where is the error.

Reply:
You can just delete this file. Ceph is already on the onprem/storage.py :)